IFC4x3/Properties/s/SeismicResistance_17fvYJ58159gRnjvJMIylv/Documentation.md

```diff
@@ -0,0 +1 @@
+Maximum level of earthquake that the equipment complies with
```

Comment:
Would probably make sense to be a bit more specific. Which scales can be used for example. "level of earthquake" is very ambiguous.
```suggestion
Maximum level of earthquake that the equipment complies with.
```

Reply:
same as sergej, if you know the measurement type we can either select the appropriate one or add it to the list in #336